warehouse/templates/manage/settings.html

```diff
@@ -98,6 +98,21 @@ <h3>{% trans %}<abbr title="two factor authentication">2FA</abbr> requirement{%
     </form>
   {% endif %}
 
+  <h3>{% trans %}OpenID Connect publishers{% endtrans %}</h3>
+  <p>
+    {% trans %}
+    OpenID Connect publishers provide a credential-free way to authenticate when uploading packages to PyPI.
+    {% endtrans %}
+  </p>
+  <!-- if OIDC is enabled for project -->
+  <p>
+    {% trans href=request.route_path('manage.project.settings.oidc', project_name=project.name), project_name=project.name %}
+    <a href="{{ href }}" class="button button--primary">
+      Manage OpenID Connect publishers for {{ project_name }}
+    </a>
+    {% endtrans %}
+  </p>
+
```

Comment:
I think this can probably just exist as a new menu option in the sidebar and the "Publishers" page and doesn't need to be here.